pkg/network/driver_buffer.go

```diff
@@ -43,6 +43,11 @@ func (d *DriverBuffer) Len() int {
 	return d.off
 }
 
+// Capacity returns the current capacity of the buffer
+func (d *DriverBuffer) Capacity() int {
+	return len(d.buf)
```

Comment:
Should this be `cap(d.buf)` ?

Reply:
In this case, no. This buffer wrapper returns existing elements from a slice, so it needs to be `len`.